tests/test_convenience.py

```diff
@@ -290,48 +291,35 @@ def test_different_unit_handling_multiple_timeseries_overrides(
                 if c >= 2030:
                     of = 0
                 else:
-                    of = harm_year_offset * (2030 - c) / (2030 - harmonisation_year)
+                    of = harm_year_offset * (2030 - c) / (2030 - year)
 
                 exp.loc[r, c] += of
+    # new requirement - we won't provide data before harmonization year
```

Comment:
Does this have to be a requirement? You can do some fun stuff with (admittedly non-emissions) harmonisation where you get things converging at a point and then spitting back out again without it (and it wasn't clear to me that having this requirement changes anything).

Reply:
Strictly - no. But I would argue first-class use cases for harmonization (as stated in the literature, etc.) are about harmonizing future values to historical values.

Practically - there is some processing code in the `aneris` internals that enforce this that I didn't have time to update robustly, hence keeping the requirement.

For the future - I would be happy to add something like a `backcast=True` kwarg that does exactly what you suggest. And contributions are, as always, most welcome =)

Reply:
FWIW, it could be that only this line needs to be adjusted: https://github.com/iiasa/aneris/blob/main/aneris/harmonize.py#L252

I just didn't have time to check